owners/src/owners_bot.js

```diff
@@ -125,6 +129,34 @@ class OwnersBot {
     await this.runOwnersCheck(github, pr);
   }
 
+  /**
+   * Adds a comment tagging always-notify owners of changed files.
+   *
+   * @param {!GitHub} github GitHub API interface.
+   * @param {!number} prNumber pull request number.
+   * @param {!FileTreeMap} fileTreeMap map from filenames to ownership subtrees.
+   */
+  async createNotifications(github, prNumber, fileTreeMap) {
+    const botComments = await github.getBotComments(prNumber);
+    if (botComments.length) {
```

Comment:
What if a file was added to a PR? Wouldn't you have to modify the existing bot comment?

Reply:
Adding a TODO on that for now, would like to address that case (along with cases where files are removed or the OWNERS file changes) separately

Reply:
This comment by @estherkim and the one above it raise good points. In both cases, I think updating the same comment will be cleaner, more accurate, and avoid comment spam. (For example, large PRs that gradually end up touching several files.)